packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -1,18 +1,139 @@
+import { ECIES, PhoneNumberUtils, SignatureUtils } from '@celo/utils'
 import BigNumber from 'bignumber.js'
+import { zip } from 'lodash'
+import * as Web3Utils from 'web3-utils'
 import { Attestations } from '../generated/types/Attestations'
-import { BaseWrapper, proxySend } from './BaseWrapper'
+import { BaseWrapper, proxyCall, proxySend, toNumber, tupleParser, wrapSend } from './BaseWrapper'
+import { CeloToken } from './TokenWrapper'
+const parseSignature = SignatureUtils.parseSignature
 
 export interface AttestationStat {
   completed: number
   total: number
 }
 
+export interface IssuerAttestationState {
+  [issuer: string]: {}
+}
+
+export enum AttestationState {
+  None,
+  Incomplete,
+  Complete,
+}
+
+export interface ActionableAttestation {
+  issuer: string
+  attestationState: AttestationState
+  time: number
+  publicKey: string
+}
+
+const parseAttestationInfo = (rawState: { 0: string; 1: string }) => ({
+  attestationState: parseInt(rawState[0], 10),
+  time: parseInt(rawState[1], 10),
+})
+
+function attestationMessageToSign(phoneHash: string, account: string) {
+  const messageHash: string = Web3Utils.soliditySha3(
+    { type: 'bytes32', value: phoneHash },
+    { type: 'address', value: account }
+  )
+  return messageHash
+}
+
 export class AttestationsWrapper extends BaseWrapper<Attestations> {
+  attestationExpirySeconds = proxyCall(
```

Comment:
We should document every public method, and mark as private the rest.

Also, I've found that when using an input parser, the final input parameter names don't express nothing. So, in that case we should also make the property type explicit. See Exchange wrapper for examples:

```ts
/**
   * @dev Returns the amount of buyToken a user would get for sellAmount of sellToken
   * @param sellAmount The amount of sellToken the user is selling to the exchange
   * @param sellGold `true` if gold is the sell token
   * @return The corresponding buyToken amount.
   */
  getBuyTokenAmount: (sellAmount: NumberLike, sellGold: boolean) => Promise<BigNumber> = proxyCall(
    this.contract.methods.getBuyTokenAmount,
    tupleParser(parseNumber, identity),
    toBigNumber
  )
```